cranelift/codegen/src/ir/dfg.rs

```diff
@@ -37,19 +37,19 @@ pub struct DataFlowGraph {
     /// Data about all of the instructions in the function, including opcodes and operands.
     /// The instructions in this map are not in program order. That is tracked by `Layout`, along
     /// with the block containing each instruction.
-    insts: PrimaryMap<Inst, InstructionData>,
+    pub(crate) insts: PrimaryMap<Inst, InstructionData>,
```

Comment:
I believe you can access all these through the public api's. I did prefer that over direct accesses as it is less likely to violate invariants.